client/main.go

```diff
@@ -239,6 +253,36 @@ func handleTermboxEvent(ev termbox.Event, conn *websocket.Conn) error {
 		switch ev.Key {
 		case termbox.KeyEsc, termbox.KeyCtrlC:
 			return errors.New("rowix: exiting")
+		case termbox.KeyCtrlS:
+			if fileName != "" {
+				crdt.Save(fileName, &doc)
+				e.StatusMsg = "Saved document to " + fileName
+				e.SetStatusBar()
+			} else {
+				e.StatusMsg = "No file to save to!"
+				e.SetStatusBar()
+			}
+		case termbox.KeyCtrlL:
+			if fileName != "" {
+				logger.Log(logrus.InfoLevel, "LOADING DOCUMENT")
+				newDoc, err := crdt.Load(fileName)
+				e.StatusMsg = "Loading " + fileName
+				e.SetStatusBar()
+				if err != nil {
+					e.StatusMsg = "Failed to load " + fileName
+					logrus.Error("failed to load file %s", fileName)
```

Comment:
tiny change: `logrus.Errorf`

Reply:
Ah thanks! Don't know why I missed that.